asg/src/expression/circuit_init.rs

```diff
@@ -70,8 +70,17 @@ impl<'a> ExpressionNode<'a> for CircuitInitExpression<'a> {
         true
     }
 
-    fn const_value(&self) -> Option<ConstValue> {
-        None
+    fn const_value(&self) -> Option<ConstValue<'a>> {
+        let mut members = IndexMap::new();
+        for (identifier, member) in self.values.iter() {
+            // insert by name because accessmembers identifiers are different.
+            members.insert(
```

Comment:
can you assert(debug `assert!` should be fine) that there is no overwrite here? should already be prevented, but just to be sure